lexus_rx_450h_2019/SubsystemControllerParams.yaml

```diff
@@ -18,6 +18,13 @@ hardware_interface:
       subsystem_namespace: /hardware_interface
       required_subsystem_nodes: ['']
       unmanaged_required_nodes: ['']
+      excluded_namespace_nodes: # Lidar and GPS drivers are handled by the localization subsystem
+        - /hardware_interface/velodyne_lidar_driver_wrapper_node
+        - /hardware_interface/carma_novatel_driver_wrapper_node
+      ros1_required_drivers: 
+       - /hardware_interface/controller
```

Comment:
Can we leave this empty for the Lexus, since the SSC interface wrapper node is in ROS2? Additionally, I think the node would be `/hardware_interface/ssc_interface_wrapper_node`